example/src/modules/user.ts

```diff
@@ -23,6 +30,19 @@ const getDefaultUser = (): IUser => ({
   name: '',
 })
 
+const localStoragePlugin: IPlugin<IUser> = {
```

Comment:
It would be nice to provide a generic local storage plugin in the library, but I'm not sure about the right way to make it work automatically with submodules yet.